workflows/kneaddata_paired.smk

```diff
@@ -36,6 +36,22 @@ rule compressdata2:
     run: 
         shell("gzip -c {input} > {output}")
 
+checkpoint furthercompress:
+    input: "{kneadfolder}/{sample}*.fastq"
+    output: "{kneadfolder}/{sample}*.fastq.gz"
```

Comment:
This definitely won't work. And the input > output thing won't work for the same reason as your previous compression attempt - the redirect operator (`>`) can only take one output on the right hand side.

You may need a for loop or to glob the files in some other way and then do something like

```
rule further_compress:
	input: '{sample}_{suffix}.fastq'
	output: '{sample}_{suffix}.fastq.gz'
	shell: 'gzip {input}'
```